src/coreComponents/virtualElement/CMakeLists.txt

```diff
@@ -3,28 +3,28 @@
 #
 set( virtualElement_headers
   ConformingVirtualElementOrder1.hpp
+  ConformingVirtualElementOrder1_impl.hpp
   VirtualElementBase.hpp
   )
 #
 # Specify all sources
 #
 set( virtualElement_sources
-  ConformingVirtualElementOrder1.cpp
   )
 
-if( BUILD_OBJ_LIBS)
-  set( dependencyList dataRepository codingUtilities)
-else()
-  set( dependencyList common)
-endif()
+# if( BUILD_OBJ_LIBS)
+#   set( dependencyList dataRepository codingUtilities)
```

Comment:
You have to be dependent on something don't you? At the very least you must be dependent on `common`?

Reply:
These dependencies are commented out because I am currently not compiling anything in the VEM folder and cmake fails if I declare dependencies. I left these lines as comments for future reference.